### PR TITLE
Dlg unstack

### PR DESCRIPTION
**Pull request by Muthenya**

### CODE REVIEW

**Comment by Muthenya** [comment]
@dannyparsons please review this.

File: instat/dlgUnstack.vb [hunk 1]
```diff
@@ -85,7 +90,9 @@ Public Class dlgUnstack
         ucrPnlUnstackCol.AddRCodeIsRFunctionCondition(rdoRestoreHierarchy, bNewIsPositive:=False)
         'TODO add function name condition for rdoMultiple
 
+        ucrPnlUnstackCol.AddToLinkedControls(ucrReceiverColumnToUnstack, {rdoSingle}, bNewLinkedHideIfParameterMissing:=True)
```
Comment (dannyparsons) [inline instat/dlgUnstack.vb]: If in here you set AddRemove to True then you won't need to remove the parameter manually.

File: instat/dlgUnstack.vb [hunk 2]
```diff
@@ -193,6 +201,7 @@ Public Class dlgUnstack
             clsBaseRCode = clsUnstackedOperator
             ucrBase.clsRsyntax.SetBaseROperator(clsUnstackedOperator)
             clsDcastFunction.RemoveAssignTo()
+            clsDcastFunction.RemoveParameterByName("value.var")
```
Comment (dannyparsons) [inline instat/dlgUnstack.vb]: Would be better if this was done through linking as this could cause bugs if the parameter is not added back in when needed.

**Comment by Muthenya** [comment]
@dannyparsons I have now made the changes.

**Pull request merged**